recipes/dav1d/all/conanfile.py

```diff
@@ -0,0 +1,104 @@
+import os
+from conans import ConanFile, Meson, tools
+
+required_conan_version = ">=1.29.1"
+
+class Dav1dConan(ConanFile):
+    name = "dav1d"
+    description = "dav1d is a new AV1 cross-platform decoder, open-source, and focused on speed, size and correctness."
+    homepage = "https://www.videolan.org/projects/dav1d.html"
+    topics = ("conan", "av1", "codec", "video", "encoding")
+    license = "BSD-2-Clause"
+    url = "https://github.com/conan-io/conan-center-index"
+    options = {
+        "shared": [True, False],
+        "fPIC": [True, False],
+        "bit_depth": ["all", 8, 16],
+        "with_tools": [True, False],
+        "assembly": [True, False],
+        "with_avx512": [True, False],
+    }
+    default_options = {
+        "shared": False,
+        "fPIC": True,
+        "bit_depth": "all",
+        "with_tools": True,
+        "assembly": False,
```

Comment:
shouldn't it be True by default? https://github.com/videolan/dav1d/blob/6ed5fafb42c651c24b6a65fd4f50ed426fd72d65/meson_options.txt#L10

Reply:
I prefer to disable options that make non-portable binaries that can build on CI but would not work on client machines without rebuilding. For this library, we can enable assembly by default (I found no autodetect) while keeping avx512 off. How do you think?

Reply:
Agree, exactly what I was thinking 😉

Reply:
Done